### PR TITLE
[GEN_BASEADDRESS] EXCLUDE list: Sort and update it a bit

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Sort and update it a bit

## Proposed changes

- Sort it by main directories
to ease maintenance.
- Add comments about 'v0.4.14- backward-support'
Addendum to 0fb580a (0.4.15-dev-5741).
- Add the 4 other HALs
Addendum to 0a951f1 (0.4.15-dev-6823) and olders.
- Add 2 winetest 'msi_*.dll'
Follow-up to 6c93f3c (0.4.15-dev-7359).

### CODE REVIEW

File: sdk/tools/gen_baseaddress.py [hunk 1]
```diff
@@ -201,20 +193,40 @@
...
     'redirtest1.dll',
     'redirtest2.dll',
+    'testvdd.dll',
     'win32u_2k3sp2.dll',
     'win32u_vista.dll',
     'win32u_xpsp2.dll',
-    'testvdd.dll',
+    # sdk.
+    'genincdata.dll',
+    # win32ss.
```
Comment (HBelusca) [inline sdk/tools/gen_baseaddress.py:225]: Trailing dot unnecessary

Comment (SergeGautherie) [reply sdk/tools/gen_baseaddress.py:225]: Replaced by `/`.

**Comment by SergeGautherie** [comment]
Review comment addressed 5 months ago.
Looking for a new review.
Please, reopen.

**Comment by JoachimHenze** [comment]
I cannot reopen unfortunately:
image

**Pull request closed without merging**